sh_scrapy/middlewares.py

```diff
@@ -60,6 +60,8 @@ def process_request(self, request, spider):
             request.meta[HS_PARENT_ID_KEY] = request_id
 
     def process_response(self, request, response, spider):
+        if type(response).__name__ == "DummyResponse":
+            return response
```

Comment:
I am personally not a fan of checking like this, for the record, but I am OK with it, and I realize it simplifies test/CI changes.